### PR TITLE
Show StageInfo window in OAB/VAB

### PR DESCRIPTION
**Pull request by Falki-git**
This PR brings StageInfo to OAB/VAB

Release change log:
- add StageInfo window to VAB
- window positions and states are now automatically saved on scene changes
- windows no longer persist on scene changes (Flight <-> Space Center <-> VAB)
- windows automatically open when entering Flight and VAB scenes if user didn't close them previously

Dev change log:
- [x] Add StageInfoOAB window for OAB/VAB
- [x] Add MicroEntries for StageInfo in OAB
- [x] Add OAB app button for StageInfoOAB window
- [x] Add 3 message subscriptions:
  - VesselDeltaVCalculationMessage: used for StageInfoOAB window updates (lower impact than Update())
  - GameStateEnteredMessage: used for automatically setting window visibility and loading layout states
  - GameStateLeftMessage: used for automatically setting window visibility and saving layout states
- [x] Layout states are now automatically saved and loaded on scene (gamestate) changes
- [x] MainGui window (Flight) and StageInfoOAB window (OAB/Editor) are now automatically opened on entering a scene if user didn't close them previously
- [x] MainGui window now automatically hides when not in Flight
- [x] Prettify StageInfoOAB
- [x] Remove empty stages from window
- [x] Add CelestialBodies select
- [x] Adjust TWR depending on CelestialBody
- [x] Use Vacuum TWR for SLT for Bodies without Atmosphere
- [x] Adjust design of the body selection button
- [x] Add descriptions for all new entries
- [x] Annotate properly

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,4 +21,5 @@ MicroEngineerProject/[Bb]in/
 /MicroEngineerProject/.vs
 /MicroEngineerProject/.vs/*
 /Staging/CopyStagingToGame.bat
-*.bat
\ No newline at end of file
+*.bat
+*.csproj
```
Comment (just-ero) [inline .gitignore:25]: What's the reason for this addition? Project files should not be ignored.

Comment (Falki-git) [reply .gitignore:25]: Just temporary so my github desktop ignores the project files I've edited for my setup. Will remove this once PR is ready.

**Comment by Falki-git** [comment]
Progress:
StageInfo in OAB progress

**Comment by Falki-git** [comment]
@Micrologist , ready to merge this.

**Review by Micrologist — APPROVED** [review]

**Pull request merged**